test.py

```diff
@@ -0,0 +1,4 @@
+import gi
+gi.require_version('Gst', '1.0')
+from gi.repository import Gst
+print(Gst)
```

Comment:
Could you adapt this test for the test suite?
Put the following code into file ./tests/functional/test_libraries.py:
```python
@importorskip('pyexcelerate')
def test_gi_gtk_binding(pyi_builder):
    pyi_builder.test_source(
        """
        import gi
        gi.require_version('Gst', '1.0')
        from gi.repository import Gst
        print(Gst)
        """)
```